scheds/rust/scx_layered/src/main.rs

```diff
@@ -986,6 +993,39 @@ struct Layer {
     allowed_cpus: Cpumask,
 }
 
+fn resolve_cpus_pct_range(
+    cpus_range: &Option<(usize, usize)>,
+    cpus_range_pct: &Option<(usize, usize)>,
+    max_cpus: usize,
+) -> Result<(usize, usize)> {
+    match cpus_range {
+        Some(cpus_range) => match cpus_range_pct {
+            Some(_cpus_pct) => {
+                bail!("cpus_range cannot be used with cpus_pct.");
+            }
+            None => Ok(*cpus_range),
+        },
+        None => match cpus_range_pct {
+            Some(cpus_range_pct) => {
```

Comment:
Given that `util_range` is in [0, 1.0], I wonder whether it'd be better to use fraction instead of percentage. ie. `cpus_range_frac` instead of `cpus_range_pct`.

Reply:
For consistency w/ util_range, this makes sense. 

I generally don't like floats in cfg files (probably got burned by deserialization/serialization of one forever ago or something), but consistency is good and int for one but float for the other isn't consistent.